composer.json

```diff
@@ -36,6 +37,20 @@
   "replace": {
     "eoneopay/apiformats": "*"
   },
+  "scripts":{
+    "code-sniffer": "phpcs --standard=vendor/eonx-com/standards/php-code-sniffer/EoneoPay --colors --report=full src tests -p",
+    "copy-paste-detector": "phpcpd --ansi --min-lines=5 --min-tokens=70 src tests",
+    "mess-detector": "phpmd src,tests text cleancode,codesize,controversial,design,naming,unusedcode",
+    "phpstan": "phpstan analyze src tests -c phpstan.neon --ansi --level max --memory-limit=2048M",
+    "phpunit": "phpunit --configuration phpunit.xml --coverage-text",
```

Comment:
I want to see --coverage-text go away - we can use better output formats like --coverage-php that dont spam the output with useless lines

i'd also like to see a phpmd.xml file instead of just `cleancode,codesize,etc` like in payments

Reply:
Until we can analyse --coverage-php quickly/safely, I've changed it to use --coverage-text directly to a file, rather than stdout, and only during github actions.

Running `composer phpunit` locally won't generate any output/coverage files unless specifically added